### PR TITLE
updating for cites in #27

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

File: in_text/text_02_observations.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@ title: >-
   Walleye Thermal Optical Habitat Area (TOHA) of selected Minnesota lakes: 2 Water temperature observations
 
 abstract: >-
-  Observed water temperatures from 1980-2018 were compiled for 903 lakes in Minnesota (USA). These data were used
+  Observed water temperatures from 1980-2018 were compiled for 877 lakes in Minnesota (USA). These data were used
```
Comment (lindsayplatt) [inline in_text/text_02_observations.yml:5]: Should there be an explanation as to why there is a 4-lake difference between what was compiled for observed vs modeled?

**Review by lindsayplatt — APPROVED** [review]
Awesome! Getting so close to finishing this up :)

File: in_text/text_02_observations.yml (hunk 1 above)
Comment (jread-usgs) [reply in_text/text_02_observations.yml:5]: Sure I can add that. Quick reason (to you) is that it wasn't a requirement that lakes had observed temperature to be modeled in this release - other than the requirement to be modeled by PGDL. So these four lakes had everything else that was necessary for the model, but no temperature data.

Comment (lindsayplatt) [reply in_text/text_02_observations.yml:5]: Ahh OK. You don't need to add, but it is something that I was wondering as I read the description and compared the number of lakes and thought others might also be curious :)

**Pull request merged**